packages/xo-web/src/common/intl/messages.js

```diff
@@ -718,6 +718,7 @@ const messages = {
   // ----- Pool network tab -----
   pif: 'PIF',
   poolNetworkNameLabel: 'Name',
+  poolNetworkDefault: 'Default Network',
```

Comment:
We should call this *automatic network* like XenCenter: https://docs.citrix.com/en-us/xencenter/7-1/hosts-network-properties.html